conda_recipe/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set pypi_name = "marshmallow_dataclass" %}
+{% set conda_name = "marshmallow-dataclass" %}
+{% set version = "8.0.0" %}
+
+package:
+  name: "{{ conda_name }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz"
+  sha256: "cde644b272c2284f1212f0e0d25bdb9eaf7d58f1423ae82b712ba43db229d618"
```

Comment:
Where does this hash come from ? Does it need to be manually updated on every release ?

Reply:
see my comment below

Reply:
Maybe [this](https://github.com/conda-forge/staged-recipes) help (see point 2, how do I populate the hash field?)